examples/src/test/scala/simulacrum/examples/examples.scala

```diff
@@ -95,7 +95,7 @@ class Examples extends WordSpec with Matchers {
           def empty[A] = Maybe.empty[A]
           def flatMap[A, B](fa: Maybe[A])(f: A => Maybe[B]) = fa match {
             case Just(a) => f(a)
-            case e @ Empty => e
+            case e @ Empty => Empty
```

Comment:
https://github.com/scala/bug/issues/10960

Reply:
Now `-Ywarn-unused` will warn about the unused `e`. I know everyone always compiles with `-Ywarn-unused`.